### PR TITLE
zh docs for docsy site

### PR DESCRIPTION
**Pull request by sunny0826**
Rendered at: https://sunny0826.github.io/kustomize/

Move Chinese documents to `/site` and adapt docsy

Fixed some errors

/assign @Liujingfang1
/cc @pwittrock

### CODE REVIEW

**Comment by pwittrock** [comment]
@sunny0826 This is great thanks.  It looks like there are some changes to the English version of the docs unrelated to the Chinese docs migration.  Would you mind doing these as separate PRs?

File: site/content/en/guides/plugins/builtins.md [hunk 1]
```diff
@@ -287,11 +287,6 @@ commonLabels:
 
 Adds namespace to all resources
 
-```
```
Comment (sunny0826) [inline site/content/en/guides/plugins/builtins.md:290]: en version change

File: site/content/en/api-reference/kustomization/generatoroptions/_index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ description: >
 
 Additionally, generatorOptions can be set on a per resource level within each
 generator. For details on per-resource generatorOptions usage see
-[field-name-configMapGenerator] and See [field-name-secretGenerator].
+[field-name-configMapGenerator](/kustomize/api-reference/kustomization/configmapgenerator) and See [field-name-secretGenerator](/kustomize/api-reference/kustomization/secretgenerator).
```
Comment (sunny0826) [inline site/content/en/api-reference/kustomization/generatoroptions/_index.md:14]: en version change

File: site/content/en/api-reference/glossary/_index.md [hunk 3]
```diff
@@ -414,7 +414,7 @@ A chunk of code used by kustomize, but not necessarily
 compiled into kustomize, to generate and/or transform a
 kubernetes resource as part of a kustomization.
 
-Details [here](plugins).
+Details [here](../../guides/plugins).
```
Comment (sunny0826) [inline site/content/en/api-reference/glossary/_index.md:417]: en version change

File: site/content/en/_index.md [hunk 4]
```diff
@@ -4,10 +4,27 @@ linkTitle = "Kustomize"
 
 +++
 
-{{% blocks/lead color="primary" %}}
-Kustomize provides a solution for customizing Kubernetes resource configuration free from templates
-and DSLs.
-{{% /blocks/lead %}}
+{{<blocks/cover title="Kustomize" image_anchor="center" height="full" color="primary">}}
```
Comment (sunny0826) [inline site/content/en/_index.md:7]: en version change

File: site/config.toml [hunk 5]
```diff
@@ -61,6 +61,21 @@ languageName ="English"
 # Weight used for sorting.
 weight = 1
 
+[[menu.main]]
```
Comment (sunny0826) [inline site/config.toml:64]: en version change

**Comment by sunny0826** [comment]
> @sunny0826 This is great thanks. It looks like there are some changes to the English version of the docs unrelated to the Chinese docs migration. Would you mind doing these as separate PRs?

@pwittrock I have a GitHub page open because I need to provide a preview, and all my code is on the master branch. This is not easy to separate, so I'll keep the English and Chinese content separate for future contributions.

The main changes I made to the English version were to add the home page style and example links, and to add some wrong content and links. . I'll mark them in this PR so you can review them.

File: site/content/en/api-reference/glossary/_index.md [hunk 6]
```diff
@@ -414,7 +414,7 @@ A chunk of code used by kustomize, but not necessarily
...
```
Comment (sunny0826) [inline site/content/en/api-reference/glossary/_index.md:418]: en version change

File: site/content/en/api-reference/kustomization/generatoroptions/_index.md [hunk 7]
```diff
@@ -11,7 +11,7 @@ description: >
...
```
Comment (sunny0826) [inline site/content/en/api-reference/kustomization/generatoroptions/_index.md:15]: en version change

File: site/content/en/_index.md [hunk 8]
```diff
@@ -4,10 +4,27 @@ linkTitle = "Kustomize"
...
+    class="btn btn-lg btn-secondary mr-3 mb-4"
+    href="/kustomize/contributing/"
+  >
+    Contribute <i class="fas fa-pencil-alt ml-2"></i>
+  </a>
+  <div class="mx-auto mt-5">
+    {{<blocks/link-down color="light">}}
+  </div>
+</div>
+{{</blocks/cover>}}
```
Comment (sunny0826) [inline site/content/en/_index.md:27]: en version change

File: site/config.toml [hunk 9]
```diff
@@ -61,6 +61,21 @@ languageName ="English"
...
+[languages.zh]
+title = "Kustomize"
+description = "自定义 Kubernetes 配置"
+languageName ="简体中文"
+contentDir = "content/zh"
+time_format_default = "2006年01月02日"
+time_format_blog = "2006年01月02日"
+# Weight used for sorting.
+weight = 2
+
```
Comment (sunny0826) [inline site/config.toml:78]: en version change

File: site/content/en/guides/plugins/builtins.md [hunk 10]
```diff
@@ -287,11 +287,6 @@ commonLabels:
...
-namespace: my-namespace
-```
-Adds namespace to all resources
-
 ```
 namespace: my-namespace
 ```
```
Comment (sunny0826) [inline site/content/en/guides/plugins/builtins.md:292]: en version change

File: site/content/en/guides/plugins/execPluginGuidedExample.md [hunk 11]
```diff
@@ -7,7 +7,7 @@ description: >
 ---
 
 This is a (no reading allowed!) 60 second copy/paste guided
-example.  Full plugin docs [here](README.md).
+example.  Full plugin docs [here](..).
```
Comment (sunny0826) [inline site/content/en/guides/plugins/execPluginGuidedExample.md:11]: en version change

**Pull request merged**